indico/modules/events/registration/templates/display/conference_home.html

```diff
@@ -9,12 +9,24 @@
                     {% set regform = registration.registration_form %}
                     <li>
                         <a href="{{ url_for('event_registration.ticket_download', regform) }}"
+                           class="icon-file-pdf"
                            title="{% trans title=regform.title -%}Download ticket for the '{{ title }}' registration form{%- endtrans %}">
                             {% trans title=regform.title -%}
-                                {{ title }}
+                                PDF - {{ title }}
                             {%- endtrans %}
                         </a>
                     </li>
+                    {% if regform.ticket_google_wallet_enabled %}
+                    <li>
+                        <a href="{{ url_for('event_registration.ticket_google_wallet', regform) }}"
+                           class="icon-ticket"
+                           title="{% trans title=regform.title -%}Add Google Wallet for the '{{ title }}' registration form{%- endtrans %}">
+                            {% trans title=regform.title -%}
+                                Google Wallet - {{ title }}
```

Comment:
"Google Wallet" is a product name and does not need to be translated

Reply:
Should I also move the "PDF - " (few lines before) out of translation?

Reply:
yes